README.md

```diff
@@ -91,7 +91,7 @@ elasticsearch
 Set the required environment variable:
 
 ```
-export DM_ELASTICSEARCH_URL=http://localhost:9200
+export ELASTICSEARCH_HOST=http://localhost:9200
```

Comment:
What's the rationale for renaming this when so many other things still seem to have the 'DM_' prefix?

Reply:
`ELASTICSEARCH_HOST` is what FlaskElasticsearch expects. We were previously supporting both but it makes no sense to do so when we're forced to use a specific name.